app/models/user.rb

```diff
@@ -12,15 +12,17 @@ class User < ApplicationRecord
   validates :first_name, presence: true
   validates :last_name, presence: true
   validates :intent, presence: true
+  validates :gender, presence: true
 
   enum role: %w(student admin rep)
   enum intent: %w(undeclared on_campus off_campus)
+  enum gender: %w(non-binary female male)
```

Comment:
According to the data dictionary, the values we're getting from IDR are:

> Valid values are NULL (unknown), "N" (Not Declared), "F" (Female) or "M" (Male)

How should we handle "N" and NULL cases? "N" doesn't necessarily feel like non-binary to me, but you probably have a better idea about this than I do.